build/pkgs/python3/SPKG.rst

```diff
@@ -8,16 +8,15 @@ By default, Sage will try to use system's ``python3`` to set up a virtual
 environment, a.k.a. `venv <https://docs.python.org/3.10/library/venv.html>`_
 rather than building a Python 3 installation from scratch.
 
-Sage will accept versions 3.9.x to 3.10.x.
+Sage will accept versions 3.9.x to 3.11.x.
```

Comment:
_Hello @tornaria,_
There is a small thing to update.
```suggestion
Sage will accept versions 3.9.x to 3.12.x.
```
_Best regards!_

Reply:
Good catch, thanks!